packages/caravan-wallets/src/index.ts

```diff
@@ -557,6 +601,8 @@ export function ConfigAdapter({
   policyHmac?: string;
 }) {
   switch (KEYSTORE) {
+    case BITBOX:
+      throw new Error('TODO');
```

Comment:
Not really sure what to do here or if this is needed at all for BitBox.

Reply:
This is an interaction basically to do wallet registrations (but an older API). You already have that interaction covered so I think you can just remove this, or add to both to duplicate the support.

Reply:
Added support in the same way as Ledger a few lines below.